web/src/beta/components/PublishStateSwitchField/index.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import { action } from "@storybook/addon-actions";
+import { Meta, StoryObj } from "@storybook/react";
+import React, { ReactNode } from "react";
+
+import PublishStateSwitchField from ".";
+
+export default {
+  component: PublishStateSwitchField,
+} as Meta;
+
+type Story = StoryObj<typeof PublishStateSwitchField>;
+
+const Wrapper: React.FC<{ children: ReactNode }> = ({ children }) => (
+  <div style={{ width: "174px", height: "24px" }}>{children}</div>
+);
```

Comment:
I think this hides that the component has 100% width. It looks like 174px width component.
So it's good to put the raw component without `Wrapper`.